runtime/moonbase/src/lib.rs

```diff
@@ -634,6 +634,30 @@ impl pallet_author_slot_filter::Config for Runtime {
 	type PotentialAuthors = ParachainStaking;
 }
 
+/// Migration that destroys all of pallet crowdloan rewards' stored data.
+///
+/// This is included in moonbase only so that we can re-test rewards intialization with the new
+/// version of the pallet. This migration makes no attempt to recouperate already-claimed tokens
+/// and it also mints tokens for the new pot.
+///
+/// NOT FOR USE IN VALUE-BEARING NETWORKS!
+pub struct NukeRewardsStorage;
+
+impl frame_support::traits::OnRuntimeUpgrade for NukeRewardsStorage {
+	fn on_runtime_upgrade() -> Weight {
+		// Delete all crowdlaon rewards storage
+		// Supply only the pallet name, no storage item name or map key
+		// This will cause the entire pallet's storage to be deleted
+		frame_support::storage::migration::remove_storage_prefix(b"CrowdloanRewards", &[], &[]);
```

Comment:
Any idea what the ramifications of this are on storage proof size?

Reply:
I don't think we need to know what was stored there in order to delete it. So I think the storage proof will just need to contain sibling nodes down to the root of this pallet so they can calculate the correct post-state root.

I'm not that confident that I'm right, but that makes sense to me.

Reply:
Sounds correct to me too